libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -148,6 +145,11 @@ func (m *Manager) Apply(pid int) (err error) {
 			return err
 		}
 		paths[sys.Name()] = p
+
+		if err := sys.Apply(d); err != nil {
+			m.Paths = paths
```

Comment:
If we want to do it this way, where we change m.Paths gradually, why not just change the above `paths[sys.Name()]` to `m.Paths[sys.Name()]`? Or has that map been used by callers so we have to replace the reference?